### PR TITLE
Instructions clarified and paths corrected

### LINKED ISSUE #464: Make it easier for teachers to install Maxima sandbox
The[ current explanation](https://github.com/maths/moodle-qtype_stack/blob/master/doc/en/CAS/STACK-Maxima_sandbox.md) for how to set up a Maxima sandbox is not easy to understand. There is the file [sandbox.wxm](https://github.com/maths/moodle-qtype_stack/blob/master/stack/maxima/sandbox.wxm) that could in principle make it easy, but it currently has hard-coded paths that assume the user has access to a moodle server. It would be good to have a version of this file that works well for users who do not.   Michael Kallweit (@m-r-k) suggested having different versions of the file for different operating systems to make it even easier for the user. Once these files are in place, the page explaining the setup can be drastically simplified.

### PR DESCRIPTION
**Pull request by m-r-k**
Improves the sandbox setup instructions. Fixes #464

### CODE REVIEW

File: doc/en/CAS/STACK-Maxima_sandbox.md [hunk 1]
```diff
@@ -4,17 +4,21 @@ It is very useful when authoring questions to be able to test out Maxima code in
 That is to say, to run a desktop version of Maxima with the local settings and STACK specific functions loaded.  This is also used in [reporting](../Authoring/Reporting.md) and analysis of students' responses.
 To do this you will need to load your local settings, and also the libraries of Maxima functions specific to STACK.
 
-# STACK - Maxima sandbox (without access to a server)
+## Setup Maxima and wxMaxima
+
+Install Maxima (http://maxima.sourceforge.net/) and wxMaxima (https://wxmaxima-developers.github.io/wxmaxima/).
+
```
Comment (gustavdelius) [inline doc/en/CAS/STACK-Maxima_sandbox.md]: On my system (Linux Mint) in addition to installing Maxima I also had to install the maxima-share package.

File: doc/en/CAS/STACK-Maxima_sandbox.md [hunk 2]
```diff
@@ -4,17 +4,21 @@ It is very useful when authoring questions to be able to test out Maxima code in
...
 
     .../stack/maxima/
 
-Edit and use the file `.../stack/maxima/sandbox.wmx`
+In this directory open the file `sandbox.wmx` with wxMaxima and edit it to your needs.
 
-# STACK - Maxima sandbox (with access to a server)
+## STACK - Maxima sandbox (with access to a server)
 
 It is very useful when authoring questions to be able to test out Maxima code in the same environment in which STACK uses [Maxima](Maxima.md).
```
Comment (gustavdelius) [inline doc/en/CAS/STACK-Maxima_sandbox.md]: There is duplication of text here. Also, do we still need all this complicated stuff? Isn't using sandbox.wmx always the easiest solution?

File: stack/maxima/sandbox.wxm [hunk 3]
```diff
@@ -1,40 +1,43 @@
...
 STACK Sandbox
    [wxMaxima: title   end   ] */
 
+
 /* [wxMaxima: comment start ]
 This document loads the extra files needed for STACK.
 See https://github.com/maths/moodle-qtype_stack
 
-1. Download the files from github, either using git or as a zip file.
-2. Place the files somewhere they can be read, and edit the line velow to give the location.
```
Comment (gustavdelius) [inline stack/maxima/sandbox.wxm]: Is it ever necessary to change this? Does sandbox.wxm not automatically look for the files in its own directory?

File: doc/en/CAS/STACK-Maxima_sandbox.md (hunk 1 above)
Comment (sangwinc) [reply doc/en/CAS/STACK-Maxima_sandbox.md]: Gustav,
I think if we write parallel instructions on how to install maxima locally we will end up with outdated instructions which are wrong!  So, sorry this was a problem.

**Comment by sangwinc** [comment]
We need to add the path to Windows, as wxMaxima on Windows does not always know its own path!

```
maximaplatform:"win"$
stacklocation:"c:/tmp/stack/stack"$
```

File: doc/en/CAS/STACK-Maxima_sandbox.md (hunk 1 above)
Comment (gustavdelius) [reply doc/en/CAS/STACK-Maxima_sandbox.md]: @sangwinc I think it is important to point out that just installing from the link provided is not enough, but that STACK needs additional maxima packages. I agree that we should not go into details of how to install those extra packages, we just should draw the reader's attention to the fact that they are needed.

**Pull request merged**